smartsim/entity/model.py

```diff
@@ -609,3 +627,10 @@ def _check_db_objects_colo(self) -> None:
                         "file and add it to the SmartSim Model with add_script."
                     )
                     raise SSUnsupportedError(err_msg)
+
+    def dbs_list_m(self, db_identifier):
+        self.my_list_m.append(db_identifier)
+        return self.my_list_m
+
+    def dbs_in_use_m(self) -> set():
+        return self.my_list_m
```

Comment:
I'm not understanding what the _m suffix is for. Please spell it out

Reply:
removed